packages/http/src/validator/validators/body.ts

```diff
@@ -54,6 +55,30 @@ export function splitUriParams(target: string) {
   }, {});
 }
 
+export function parseMultipartFormDataParams(target: string, multipartBoundary?: string) {
+  if(!multipartBoundary) {
+    const error = "Boundary parameter for multipart/form-data is not defined or generated in the request header. Try removing manually defined content-type from your request header if it exists.";
+    return E.left<NonEmptyArray<IPrismDiagnostic>>([
+      {
+        message: error,
+        code: 415,
+        severity: DiagnosticSeverity.Error,
+      },
+    ]);
+  }
+
+  const multipart = require('parse-multipart-data');
+  const bufferBody = Buffer.from(target, "utf-8");
+
+  // the parse-multipart-data package requires that the body is passed in as a buffer, not a string
```

Comment:
See: https://github.com/nachomazzara/parse-multipart-data/issues/14#:~:text=I%20was%20having%20a%20similar%20problem%20with%20a%20different%20payload.%20I%20opened%20up%20the%20dist/%20and%20realized%20that%20the%20author%20named%20the%20body%20param%20multipartBodyBuffer.%20I%20was%20passing%20the%20body%20as%20a%20string%2C%20but%20when%20I%20changed%20to%20pass%20it%20as%20a%20Buffer%20I%20got%20results!

And also the source code: https://github.com/nachomazzara/parse-multipart-data/blob/master/src/multipart.ts#L34C80-L34C80